src/style.css

```diff
@@ -1 +1,87 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Serif:opsz,wght@8..144,400;8..144,700&family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --blue: #00acdc;
+}
+
+html,
+body,
+a,
+img,
+ul,
+li {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+body {
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header-logo {
+  margin: 10px 50px 7px;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav-links {
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.link {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  padding: 22.5px 0;
+  white-space: nowrap;
+  margin-right: 20px;
+  cursor: pointer;
+}
+
+.link:hover {
+  color: var(--blue);
+}
+
+.nav-links:last-child .link:last-child {
+  margin-right: 0;
+}
+
+.is-active {
+  color: var(--blue);
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  right: 0;
+  left: 0;
+  bottom: -0.6px;
```

Comment:
why not `0`?

Reply:
Before that, it did not want to pass tests without specifying this parameter as -0.6, but now it has passed with 0. Thank you.